python/hail/expr/tests.py

```diff
@@ -224,5 +224,496 @@ def test_str_ops(self):
         self.assertFalse(eval_expr(s5.to_boolean()))
         self.assertFalse(eval_expr(s6.to_boolean()))
 
-
+    def check_expr(self, expr, expected, expected_type):
+        self.assertEqual(expected_type, expr.dtype)
+        self.assertEqual((expected, expected_type), eval_expr_typed(expr))
+
+    def test_arithmetic(self):
```

Comment:
Should we delete the corresponding tests from `ExprSuite`?

Reply:
will do in a separate PR.